contrib/container/dev-docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
     # Use PostgreSQL as the database backend
     # Note: This can be changed to a different backend if required
     inventree-dev-db:
-        image: postgres:13
+        image: postgres:17
```

Comment:
This is a very large jump, I do not think auto-migrations work with that many versions

Reply:
This change does not actually force any change on the user.

- If they already have a working docker compose file, they will not need to change (and their db will remain pinned at `postgres:13`) 
- If they are starting a new docker setup, they will get the latest `postgres:17` image

Additionally, they  can always edit their docker compose file as it is intended to be copied outside of the InvenTree source code anyhow